docker-compose.yml

```diff
@@ -81,6 +82,12 @@ services:
       ELASTIC_SEARCH_DATA_NODES_HOSTS: "http://ccd-elasticsearch:9200"
       DATA_STORE_DEFAULT_LOG_LEVEL: INFO
       CASE_DOCUMENT_AM_URL: http://ccd-case-document-am-api:4455
+      CCD_DEFAULTPRINTURL: https://return-case-doc-ccd.nonprod.platform.hmcts.net/jurisdictions/:jid/case-types/:ctid/cases/:cid
+      ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL: "${ENABLE_ATTRIBUTE_BASED_ACCESS_CONTROL:-true}"
+      ENABLE_PSEUDO_ROLE_ASSIGNMENTS_GENERATION: "${ENABLE_PSEUDO_ROLE_ASSIGNMENTS_GENERATION:-true}"
+      ENABLE_PSEUDO_ACCESS_PROFILES_GENERATION: "${ENABLE_PSEUDO_ACCESS_PROFILES_GENERATION:-true}"
```

Comment:
is the '-' meant to be there?

Reply:
not sure Nigel, i didnt change anything in here , same as you and Sanjay initial put together . I can try removing it doesnt look right to me either

Reply:
i found many instances of this :- in the same docker compose , i would leave it as it for now . Dont think it impacts